exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/OracleDialect.kt

```diff
@@ -9,7 +9,6 @@ internal object OracleDataTypeProvider : DataTypeProvider() {
     override fun integerType(): String = "NUMBER(12)"
     override fun integerAutoincType(): String = "NUMBER(12)"
     override fun longType(): String = "NUMBER(19)"
-    override fun ulongType(): String = "UNSIGNED BIGINT"
```

Comment:
I delete it because oracle does not support UNSIGNED syntax as far as I investigated.
Sorry for my wrong implementation.